app/react/kubernetes/cluster/NodeView/NodeApplicationsDatatable/columns.tsx

```diff
@@ -52,11 +52,19 @@ export function useColumns() {
         }),
         helper.accessor('CPU', {
           header: 'CPU reservation',
```

Comment:
The accessor helps with searching and sorting.

For using a direct string, it should be the exact key of the value - above the 'Image' accessor does get the app.Image string so that you can search for it.

For more complicated searching, like here, you can use a function to return the value to be used for searching/sorting:

```suggestion
        helper.accessor((row) => row.Resource?.cpuRequest, {
```